app/components/agenda/agenda-header/agenda-actions.hbs

```diff
@@ -5,7 +5,7 @@
   data-test-agenda-header-show-options={{true}}
   as |Menu|
 >
-  {{#if this.canEditDesignAgenda}}
+  {{#if (and this.currentSession.isEditor @currentAgenda.status.isDesignAgenda)}}
```

Comment:
This one's just me being a bit sloppy and getting confused by ember autotracking, first I thought it would not work with the getter so I removed it here, then later I found out it does actually work, so I added it elsewhere. thanks for the catch